library/std/src/ffi/c_str/tests.rs

```diff
@@ -35,8 +35,8 @@ fn build_with_zero2() {
 #[test]
 fn build_with_zero3() {
```

Comment:
This test name no longer makes sense given the change below. It needs to be renamed, or removed if empty vec is already tested elsewhere.

Reply:
Sounds good, I removed it. Honestly no idea why it existed in the first place since it's checking UB.